test/k4FWCoreTest/options/CheckOutputFiles.py

```diff
@@ -58,6 +58,17 @@ def check_events(filename, number):
         raise RuntimeError("Number of events does not match expected number")
 
 
+def check_metadata(filename, keys, values):
```

Comment:
```suggestion
def check_metadata(filename, expected_metadata):
```
Could we make this one argument that is a dict with key-value pairs instead of two lists? It would make reading and maintaining these in the future a bit easier.

Reply:
Good idea :slightly_smiling_face: